### ISSUE TITLE
String resource Hello without id in Droid Runner project

### ISSUE THREAD

**Issue by c-mu** [nunit/nunit.xamarin#57]
After upgrading Xamarin to 2.2.0.45 a string resource Hello introduced by NUnit.Runner has no id assigned in the droid test project.

Workaround is to create a `values` folder in `Resources` and add a `Strings.xml` resource file containing the resource declaration:

``` xml
<resources>
  <String name="Hello">Hello</String>
</resources>
```

**Comment by ChrisMaddock** [comment]
I don't see this problem in the latest Xamarin, perhaps there's been a fix? @c-mu, @rprouse - either of you able to confirm?

**Comment by mrbiggred** [comment]
I am also having the same problem.  My steps to recreate the problem:

1. Have an existing Visual Studio 2015 solution with an Android project using Xamarin Forms 2.3.2.127.

2. Add the Android unit tests project using the template.  It will include Xamarin Forms 2.0.0.6490.

3. Write some tests.  Everything works fine.

4. Update all the projects, including the Android unit tests project, to Xamarin Forms 2.3.3.168.

5. Get the error: 'Resource.String' does not contain a definition for 'Hello'.  In error is in the Resource.Designer.cs file:

    global::Nunit.Runner.Resource.String.Hello = global::SaturdayMP.XPlugins.Notifications.Tests.Droid.Resource.String.Hello;

The work around by @c-mu worked for me with one minor fix.  Make sure the Strings tag is in lower case:

```xml
<resources>
  <string name="Hello">Hello</string>
</resources>
```
Give me a shout if you need any additional information.

**Comment by ChrisMaddock** [comment]
Thanks @mrbiggred - those repo steps were spot on. I've submitted a potential fix as #72, if you have any idea of a better solution/definite explanation as to the change that has caused this - would love to hear it. 😄

**Comment by mrbiggred** [comment]
@ChrisMaddock thank you for fixing the issue.  I don't have a better fix then yours.  Thank you.
